index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
+        </header>
+        <div class="nav-wrapper">
+          <ul>
+            <li>
+              <a class="nav-link" href="#Introduction">Introduction </a>
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#what_you_should_already_know"
+                >what you should already know</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#JavaScript_and_Java"
+                >JavaScript and Java</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#Hello_world">Hello world</a>
+              <hr />
+            </li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
```

Comment:
Please limit the width of this section so that no line of text within it spans more than 100 characters.

Reply:
looking at the code on line 43 I can't see any text